src/main/java/com/jfrog/ide/common/configuration/JfrogCliServerConfig.java

```diff
@@ -5,15 +5,18 @@
 
 import javax.net.ssl.SSLContext;
 
+import static org.apache.commons.lang3.StringUtils.isNotBlank;
+
 public class JfrogCliServerConfig implements ServerConfig {
```

Comment:
I suggest adding the following description for this class - 

This class is used for managing JFrog CLI's configuration, so that is can be used by the IDEs.